shared/src/common/enums/api/groups-api-path.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum GroupsApiPath {
+  ROOT = '/group',
+  EDIT = '/group/edit/',
+}
```

Comment:
```suggestion
enum EAMApiPath {
  ROOT = '/'
  GROUPS = '/groups',
}
```
we should handle `edit` by `PUT` method